Tests/FunctionalTests/IdentityGetUrlVariablesTests.swift

```diff
@@ -44,12 +38,49 @@ class IdentityGetUrlVariablesTest: XCTestCase {
 
         // Clear persisted data
         UserDefaults.clear()
+        FileManager.default.clearCache()
+    }
+
+    func waitForProcessing(interval: TimeInterval = 0.5) {
+        let expectation = XCTestExpectation()
+        DispatchQueue.global().asyncAfter(deadline: DispatchTime.now() + interval - 0.1) {
+            expectation.fulfill()
+        }
+        wait(for: [expectation], timeout: interval)
+    }
+
+    func registerEdgeIdentityAndStart() {
```

Comment:
Since these are helper functions for this test, I suggest we make these private and we move them at the end of the file.